### PR TITLE
update useAvatar params

### PR DESCRIPTION
**Pull request by mdtanrikulu**
Updating avatar endpoint response parameters, based on json type changes here and network parameter addition in here.

### CODE REVIEW

**Comment by makoto** [comment]
Screenshot 2021-09-15 at 20 09 41

Top left image is broken.  

You need to update `NetworkInformation.js` and remove `imageUrl` if you are not using it.

**Comment by makoto** [comment]
When I upload image url, this time it shows the top left okay but not at the avatar text record section.
Screenshot 2021-09-15 at 20 17 59

File: src/components/hooks.js [hunk 1]
```diff
@@ -17,6 +16,15 @@ export function useScrollTo(pos) {
   }, [pos])
 }
 
+const networkName = {
+  main: 'mainnet',
+  goerli: 'goerli',
+  kovan: 'kovan',
```
Comment (makoto) [inline src/components/hooks.js:22]: We don't support kovan

**Comment by mdtanrikulu** [comment]
> When I upload image url, this time it shows the top left okay but not at the avatar text record section.
> Screenshot 2021-09-15 at 20 17 59

I didn't change the functionality of implementation. In previous one, condition for running avatar method was [this](https://github.com/ensdomains/ens-app/blob/db4bd8a678954eb9429cceefa724ef420565c330/src/components/hooks.js#L240) -> `uri?.match(/^eip/)` and it is same in this PR as well. If you think we should show images for other protocols (http, ipfs e.g.) I can extend to it.

**Comment by makoto** [comment]
Yes, it would be great if you can show other protocol images according to the specification https://gist.github.com/Arachnid/9db60bd75277969ee1689c8742b75182

**Comment by carlosdp** [comment]
Can we also add `ar://` support, as in #1266 ?

**Comment by carlosdp** [comment]
That said, I would advocate for #1266 as it eliminates the need for the server-side metadata endpoint for avatar images and depends entirely on decentralized gateways, including support for the NFT `eip155/erc721` reference.

**Review by makoto — APPROVED** [review]

**Pull request merged**